seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,7 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.4\\s*\\/\\s*2\\.0/.test(code), 'message: You should use the <code>/</code> operator');"
```

Comment:
I agree and that makes sense. Better to make the tests as bulletproof as possible with each update. I'll take a crack at this later today.